rnnoise/signal/README.md

```diff
@@ -0,0 +1,16 @@
+### How to build the `signal` WASM files?
+Download and install [Emscripten](https://emscripten.org/) follow the [guide](https://emscripten.org/docs/getting_started/downloads.html). Emscripten compiles C and C++ to WebAssembly using LLVM and Binaryen. Emscripten output can run on the Web, in Node.js, and in wasm runtimes. 
+
+Download [RNNoise](https://github.com/xiph/rnnoise) and enter the `examples` folder. Extract the pre-processed and post-processed parts of the project and write the code following the `rnnoise_demo.c`.
+
+Generate the WASM file:
+```
+emcc -g -O3 -s ALLOW_MEMORY_GROWTH=1 -s EXPORT_ALL=1 -s EXPORTED_RUNTIME_METHODS='["cwrap"]' -I ../include/ ../src/*.c ../examples/your_code.c -o signal.js
```

Comment:
What is the `your_code.c`? Can developers reproduce the signal.js by following your steps?

Reply:
The `your_code.c` is the pre-processed and post-processed parts of the RNNoise project and which is written follow the `rnnoise_demo.c`. BTW, I think we only need to tell the developer how it came from and how to use it, otherwise we need to maintain a modified RNNoise project somewhere. How do you think about it?

Reply:
The developers should be able to check out the source code and follow your steps to re-compile the `signal.js`. This is a sample code for tutorial, not only a demo.

Reply:
> The developers should be able to check out the source code and follow your steps to re-compile the `signal.js`. This is a sample code for tutorial, not only a demo.

I understand what you mean, then I upload the files that need to be modified to the current directory and let the developers follow the README to add or replace the corresponding files to compile?

Reply:
As the implementation is based on https://github.com/xiph/rnnoise, probably you can host your modification in your forked repo and post a pointer in this README.md. You can also send your modification as PR to the original repo and get the feedback from the author of rnnoise.

Reply:
I have hosted my modification in the forked repo and introduced the generation method of wasm files in detail.